Rules/CTF/mapcycle.cfg

```diff
@@ -3,52 +3,51 @@
 
 mapcycle =
 				#8x
-			Maps/Official/CTF/8x_Fest_Pass.png;
 			Maps/Official/CTF/8x_BaldJoyce.png;
 			Maps/Official/CTF/8x_BBiscuit.png;
 			Maps/Official/CTF/8x_SnitchMerc.png;
-			Maps/Official/CTF/8x_Gloryhill2.png;
+			Maps/Official/CTF/8x_Gloryhill.png;
 				#bunnie
 			Maps/Official/CTF/bunnie_Elantris.png;
-			Maps/Official/CTF/bunnie_Luthadel.png;
 			Maps/Official/CTF/bunnie_MetalOopa.png;
 			Maps/Official/CTF/bunnie_MrBraun.png;
 			Maps/Official/CTF/bunnie_ShiinaKagari.png;
 			Maps/Official/CTF/bunnie_AkihaRumiho.png;
 			Maps/Official/CTF/bunnie_OkabeRintaro.png;
 			Maps/Official/CTF/bunnie_AlexisLeskinen.png;
-			Maps/Official/CTF/bunnie_the-End.png;
 			Maps/Official/CTF/bunnie_Copied-City.png;
 			Maps/Official/CTF/bunnie_meaningless-Code.png;
 			Maps/Official/CTF/bunnie_or-not-to-Be.png;
-			Maps/Official/CTF/bunnie_Elric.png
+			Maps/Official/CTF/bunnie_Elric.png;
+			Maps/Official/CTF/bunnie_Roy.png;
+			Maps/Official/CTF/bunnie_Vague-Hope.png;
 				#Geti
 			Maps/Official/CTF/Snatchmark.png;
 			Maps/Official/CTF/CerberosRift.png;
 			Maps/Official/CTF/HearthPlains.png;
+			Maps/Official/CTF/NewTortuga.png;
 				#Rayne
 			Maps/Official/CTF/Rayne_KingArthursPonds.png;
-				#Oodle
-			Maps/Official/CTF/Oodle_Stonethrow.png;
 				#JTG
 			Maps/Official/CTF/JTG_FirstBadlands.png;
 				#Magmus
 			Maps/Official/CTF/Magmus_Holegmus.png;
 			Maps/Official/CTF/Magmus_hangingmus.png;
 				#mcrifel
 			Maps/Official/CTF/mcrifel_Steppes.png;
+			Maps/Official/CTF/mcrifel_fish.png
 				#Ferrezinhre
 			Maps/Official/CTF/Ferrezinhre_Rost.png;
-			Maps/Official/CTF/Ferrezinhre_Asuron.png;
-			Maps/Official/CTF/Ferrezinhre_Dulvaren.png;
 			Maps/Official/CTF/Ferrezinhre_Avinaaq.png;
 			Maps/Official/CTF/Ferrezinhre_Voltaric.png;
+			Maps/Official/CTF/Ferrezinhre_Echo_No.2_Replica.png;
```

Comment:
Has the map name been tested not to break anything UI/server loading/whatever? I would rather avoid having file names with extraneous dots even if it appears to work for now.